src/points/saga.ts

```diff
@@ -296,21 +298,57 @@ export function* sendPendingPointsEvents() {
   }
 }
 
-function* watchGetHistory() {
+export function* watchSwapSuccessTransformPayload({
```

Comment:
This pattern is really just for testing purposes, and I don't really like it... I was able to inline this (see [here](https://github.com/valora-inc/wallet/pull/5643/commits/77b808b5bde5466f5a7a9c53655bc9c3c56718aa#diff-4187b1512dcd56a9389e0b7054cdb54b30a8e632563003e5c7068bae36b817adR317)), though I couldn't get the expect-saga matchers to match the `sendPointsEvent` call, likely related to the inlining.